tests/cupyx_tests/scipy_tests/ndimage_tests/test_interpolation.py

```diff
@@ -55,6 +55,13 @@ def test_map_coordinates_float(self, xp, scp, dtype):
         coordinates = testing.shaped_random((a.ndim, 100), xp, dtype)
         return self._map_coordinates(xp, scp, a, coordinates)
 
+    @testing.for_float_dtypes(no_float16=True)
+    @testing.numpy_cupy_allclose(atol=1e-5, scipy_name='scp')
+    def test_map_coordinates_float_nd_coords(self, xp, scp, dtype):
+        a = testing.shaped_random((100, 100), xp, dtype)
+        coordinates = testing.shaped_random((a.ndim, 10, 10), xp, dtype, scale=99.0)
```

Comment:
One thing remained. Would you break this line so that it passes flake8?

```suggestion
        coordinates = testing.shaped_random(
            (a.ndim, 10, 10), xp, dtype, scale=99.0)
```